dll/shellext/netplwiz/netplwiz.rc

```diff
@@ -0,0 +1,26 @@
+#include <windef.h>
+#include <winuser.h>
+#include <commctrl.h>
+
+#include "resource.h"
+
+LANGUAGE LANG_NEUTRAL, SUBLANG_NEUTRAL
+
+#define REACTOS_VERSION_DLL
+#define REACTOS_STR_FILE_DESCRIPTION  "Map Network Drives"
+#define REACTOS_STR_INTERNAL_NAME     "netplwiz.dll"
+#define REACTOS_STR_ORIGINAL_FILENAME "netplwiz.dll"
+
+#include <reactos/version.rc>
+
+#include <reactos/manifest_hosted.rc>
+
+/* Icons */
+IDI_DISCONNECT_NET_DRIVES ICON "res/4400.ico"
```

Comment:
Why is the icon named 
![image](https://user-images.githubusercontent.com/1969829/56612362-79c76300-6614-11e9-9e81-6f066a157c40.png)
??

Reply:
That's what Windows had it define as.